docs/source/json-api/index.rst

```diff
@@ -92,21 +92,22 @@ using.
 With Query Store
 ------------------
 
-In production setups, you should configure the JSON API to use a
-PostgreSQL backend as a cache. The in-memory backend will call the
+You can configure the JSON API to use a PostgreSQL/Oracle(Enterprise Edition Only)
+backend as a cache. The in-memory backend will call the
 ledger to fetch the entire active contract set for the templates in
 your query every time so it is generally not recommended to rely on
 this in production. Note that the PostgreSQL backend acts purely as a
 cache. It is safe to reinitialize the database at any time.
 
 To enable the PostgreSQL backend you can use the ``--query-store-jdbc-config`` flag, an example of which is below.
 
-.. note:: When you use the Query Store you'll want your first run to specify ``createSchema=true`` so that all the necessary tables are created. After the first run make sure ``createSchema=false`` so that it doesn't attempt to create the tables again.
+.. note:: For development purposes we suggest to use  ``start-mode=create-if-needed-and-start`` so that all the necessary tables are created if they don't exist.
```

Comment:
The change is relevant since `createSchema=true` is no longer there IIRC, but I wouldn't suggest to use the query store "for development purposes".

Reply:
ok removed the wording "for development purposes" , my idea was to suggest using `create-if-needed-and-start` here and the relevant `create-only, start-only` modes in production setups where we want separation of credentials.

Reply:
I'm not aware that _we_ want that separation of credentials. We've been asked to support it, but that's not quite the same as presenting it as the only way to run in production.